src/components/users.vue

```diff
@@ -1,135 +1,142 @@
-              v-for="application in user.applications"
-              :key="application.id"
-            >
-              {{ application.type }}
-            </span>
-          </td>
-          <td>{{ getMostRecentApplicationDate(user) }}</td>
-        </tr>
-      </tbody>
-    </table>
   </div>
 </template>
 
 <script>
+  import localforage from '../sessionUtils'
   import axios from 'axios';
   import sortableTh from '@/components/sortableTh';
+  import _ from 'lodash';
 
   const defaultSortAscending = true;
   const sortColumns = ['id', 'name', 'email', 'roles', 'date']
   const roles = ['admin', 'parent', 'sponsor', 'volunteer'];
 
   export default {
-    name: 'users',
-    components: {
-      'sortable-th': sortableTh,
-    },
-    data: function() {
+    data(){
       return {
-        filterRole: "",
-        roles: roles,
-        sortAscending: defaultSortAscending,
-        sortColumn: "id",
-        sortColumns: sortColumns,
+        filterString: '',
+        filteredUsers: [],
         users: [],
+        columns: ['name', 'email', 'registered', 'role'],
+        status: {
+          Name: 'fa fa-angle-right',
+          Email: 'fa fa-angle-right',
+          Registered: 'fa fa-angle-down',
+          Role: 'fa fa-angle-right'
+        }
       }
     },
     computed: {
-      sortedUsers: function() {
-        return this.filteredUsers.sort((left, right) => {
-          let leftColumn = null;
-          let rightColumn = null;
-          switch (this.sortColumn) {
-            case 'date':
-              leftColumn = this.getMostRecentApplicationDate(left);
-              rightColumn = this.getMostRecentApplicationDate(right);
-              break;
-            case 'roles':
-              leftColumn = left.applications.length;
-              rightColumn = right.applications.length;
-              break;
-            default:
-              leftColumn = left[this.sortColumn];
-              rightColumn = right[this.sortColumn];
+      filterUsers: function() {
+        this.filteredUsers = this.users.map(u => {
+          let date =  new Date(parseInt(u._id.$oid.substring(0, 8), 16) * 1000);
+          let shortDate = '' + date
+          u._id.$oid = shortDate.slice(4,15)
+          if (u.full_name.toLowerCase().includes(this.filterString.toLowerCase()) || u.email.toLowerCase().includes(this.filterString.toLowerCase())) {
+            return u;
           }
-
-          let sort = 0;
-          if (leftColumn > rightColumn) {
-            sort = 1;
-          } else if(leftColumn < rightColumn) {
-            sort = -1;
-          }
-
-          return (this.sortAscending ? sort : -sort);
-        });
-      },
-      filteredUsers: function() {
-        return this.users.filter(user => {
-          let userRoles = user.applications.map(application => { return application.type; });
-          return this.filterRole === "" || userRoles.includes(this.filterRole);
         });
       },
+      testing: function() {
```

Comment:
I don't see where this function is used, unless I missed it somewhere. If it's never called, I say we get rid of it.